UnityProject/Assets/Scripts/Health/Sickness/Sickness.cs

```diff
@@ -33,16 +35,21 @@ public class Sickness : MonoBehaviour
 		private int currentStage = 1;
 		public int CurrentStage => currentStage;
 
-		[FormerlySerializedAs("possibleCures")]public List<Chemistry.Reagent> PossibleCures = new List<Chemistry.Reagent>();
+		[FormerlySerializedAs("possibleCures")]public List<Reaction> PossibleCures = new List<Reaction>();
 		public List<PlayerHealthData> ImmuneRaces = new List<PlayerHealthData>();
-		public Chemistry.Reagent CureForSickness = null;
-		public List<Chemistry.Reagent> CureHints = new List<Chemistry.Reagent>();
+		public Reagent CureForSickness = null;
+		public List<Reagent> CureHints = new List<Chemistry.Reagent>();
 
 		[SerializeField, Tooltip("basic Symptomp feedback")] protected EmoteSO emoteFeedback;
 
 		[SerializeField, Range(10f,60f)] private float cooldownTime = 10f;
 		public bool IsOnCooldown = false;
 
+		public LayerMask PlayerMask;
+
+		private int cureIndex = 0;
+		public int CureIndex => cureIndex;
```

Comment:
Is there a reason to provide the index here instead of the `Reaction` directly?

Reply:
we want to remember which index we chosen from the possible cure list